src/index.js

```diff
@@ -0,0 +1,223 @@
+    },
+  },
+  {
+    //drop down of license options
+    type: "list",
+    message: "Please select the license for your project",
+    name: "projectLicense",
+    choices: [
+      "Apache",
+      "GNU General Public",
+      "MIT",
+      "BSD 2-Clause Simplified",
+      "BSD 3-Clause",
+      "Boost Software",
+      "Creative Commons Zero",
+      "Eclipse Public 2.0",
+      "None",
+    ],
+  },
+  {
+    //conditional
+    // when: "confirmContribution",
+    type: "confirm",
+    message: "Would you like to enter contribution guidelines for the project?",
+    name: "confirmContribution",
+  },
+  {
+    //conditional
+    // when: "confirmContribution",
+    type: "input",
+    message: "Please enter contribution guidelines for the project",
+    name: "projectContribution",
+    when(answers) {
+      return answers.confirmContribution;
+    },
+  },
+  {
+    //conditional
+    // when: "confirmUsage",
+    type: "confirm",
+    message: "Would you like to add usage information for the project?",
+    name: "confirmUsage",
+  },
+  {
+    //conditional
+    // when: "confirmUsage",
+    type: "input",
+    message: "Please enter usage information for the project",
+    name: "projectUsage",
+    when(answers) {
+      return answers.confirmUsage;
+    },
+  },
+  {
+    //conditional
+    // when: "confirmTests",
+    type: "confirm",
+    message:
+      "Would you like to add information on how a user could test the project?",
+    name: "confirmTests",
+  },
+  {
+    //conditional
+    // when: "confirmTests",
+    type: "input",
+    message: "Please enter information on how a user could test the project:",
+    name: "projectTests",
+    when(answers) {
+      return answers.confirmTests;
+    },
+  },
+  {
+    type: "input",
+    message: "What is your github username?",
+    name: "projectUsername",
+    validate(answers) {
+      if (!answers) {
+        return "Please enter a your github username.";
+      }
+      return true;
+    },
+  },
+  {
+    type: "input",
+    message: "What is your email address?",
+    name: "projectEmail",
+    validate(answers) {
+      if (!answers) {
+        return "Please enter your email address.";
+      }
+      return true;
+    },
+  },
+];
+
+const getBadge = (license) => {
+  if (license === "None") return "";
+
+  let badgeUrl;
+
+  switch (license) {
+    case "MIT":
+      badgeUrl = "https://img.shields.io/badge/MIT-license-green";
+      break;
+
+    case "Apache":
+      badgeUrl = "https://img.shields.io/badge/Apache-license-green";
+      break;
+
+    default:
+      break;
+  }
+  return `![${license} badge](${badgeUrl})`;
+};
+
+const generateReadMe = (answers) => {
+  const contributionsText = answers.confirmContribution
+    ? `The following contributions were made: ${answers.projectContribution}`
+    : "No project contributions";
+  return `# ${answers.projectTitle} ${getBadge(answers.projectLicense)}
+
+    ## Table of Contents
```

Comment:
Really annoying but the only reason this is broken is because it's tabbed in. If you highlight everything from the `## Table of Contents` line until the github profile line and do command + shift + tab twice (to reverse tab), it will fix it